pages/exhibition/book-your-stand.jsx

```diff
@@ -1,5 +1,5 @@
 import { Container } from "@mui/material";
-
+import {SectionTitle} from "../../src/components/StyledTypography";
```

Comment:
you don't have to use relative imports
in our project we can do global imports directly
change this to :
import {SectionTitle} from "components/StyledTypography"

this will make copying imports between file easier

Reply:
yeah, I tried it and it works perfect. seems like it is nextjs feature. I familiarize myself with it. it is more easy.

Reply:
you can configure normal "create react app" project to use it as well
it's a good practice